test/celeritas/em/MuBremsstrahlung.test.cc

```diff
@@ -161,41 +189,82 @@ TEST_F(MuBremsstrahlungInteractorTest, stress_test)
                 {
                     Interaction result = interact(rng_engine);
-                    // SCOPED_TRACE(result);
                     this->sanity_check(result);
+
+                    energy += result.secondaries.front().energy.value();
+                    costheta += dot_product(
+                        result.direction, result.secondaries.front().direction);
                 }
                 EXPECT_EQ(num_samples,
                           this->secondary_allocator().get().size());
                 num_particles_sampled += num_samples;
             }
-            avg_engine_samples.push_back(double(rng_engine.count())
-                                         / double(num_particles_sampled));
+            avg_engine_samples.push_back(real_type(rng_engine.count())
+                                         / num_particles_sampled);
+            avg_energy.push_back(energy / num_particles_sampled);
+            avg_costheta.push_back(costheta / num_particles_sampled);
         }
     }
 
     // Gold values for average number of calls to RNG
-    double const expected_avg_engine_samples[] = {10.4316,
-                                                  9.7148,
-                                                  9.2378,
-                                                  8.6495,
-                                                  8.5108,
-                                                  10.4121,
-                                                  9.7221,
-                                                  9.2726,
-                                                  8.6439,
-                                                  8.5178};
+    static double const expected_avg_engine_samples[] = {8.1181,
+                                                         8.4645,
+                                                         9.089,
+                                                         10.2864,
+                                                         8.6038,
+                                                         8.1621,
+                                                         8.1053,
+                                                         8.1199,
+                                                         8.4714,
+                                                         9.0945,
+                                                         10.259,
+                                                         8.5959,
+                                                         8.1658,
+                                                         8.1053};
+    static double const expected_avg_energy[] = {0.0038567895491616,
+                                                 0.019186468482337,
+                                                 0.098553972746449,
+                                                 1.6852671846735,
+                                                 229.53138001979,
```

Comment:
Okay, this average number seems much more reasonable!  A good additional test may be a comparison of the transverse momentum along the line of flight between gamma and muon after interaction, which should be same. This is just for a note, so it is okay without the test as we will validate angular distribution too.